README.md

```diff
@@ -1,3 +1,5 @@
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/kubesphere/s2i-java-container)
```

Comment:
```diff
-kubesphere/s2i-java-container
+kubesphere/s2i-nodejs-container
```

Reply:
My bad. Good review.

Reply:
This repo does not have the squash merge method setting. So, I just push the commit in the force way.